lib/billing.rb

```diff
@@ -0,0 +1,24 @@
+class Billing
+
+  def sum(order)
```

Comment:
this method can be refactored to look cleaner:
something akin to:
```ruby
order.map { |item| item[:price] * item[:quantity] if item.key?(:name) }.compact.reduce(:+)
```
Without knowing why you're doing the things here, I would still split this up into three different methods to reduce the number of operations this method is doing: mapping, checking a conditional, looping and summing.

If that makes sense?